assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,157 @@
 console.log('***** Music Collection *****')
+
+// Update the `3-music-collection.js` file to do the following:
+
+// > While working through the features below, make sure to commit after you complete a task! Your commit message should reflect what you added or changed since the previous commit.
+
+// ### Required Features
+
+// - Create a variable `collection` that starts as an empty array.
+
+let collection = [];
+
+// - Add a function named `addToCollection`. This function should:
+//   - Take in the album's `title`, `artist`, `yearPublished` as input parameters
+//   - Create a new object having the above properties
+//   - Add the new object to the end of the `collection` array
+//   - Return the newly created object
+
+function addToCollection(titleName, artistName, yearPublished) {
+    let newObject = {
```

Comment:
As you get more comfortable with the core syntax, you can start thinking about some of the **really hard** programming problems. Problems like _naming variables_.

General tip: make variable names as specific as they can be. Names like "object", "array", "string" are super generic, and probably only appropriate in functions that are equally generic (i.e. a function like `map` which is designed to work with _any_ array).

In this case, a good name might have been "album".